### ISSUE TITLE
Namespace prefixes ending in '/' not applied on output of EYE

### ISSUE THREAD

**Issue by sanderzz91** [josd/eye#5]
Hi,

I would like to request a minor improvement to EYE.

It seems that in the output of EYE, URLs are abbreviated using a namespace prefix, if that prefix is ending with '#'. However, this abbreviation is **not** applied when the URL's namespace prefix ends with '/'.

For example:

```
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX dcterms: <http://purl.org/dc/terms/>

<http://127.0.0.1:8080/plugins/organization-tree> rdfs:label "Organization tree"@en.
<http://127.0.0.1:8080/plugins/organization-tree> <http://purl.org/dc/terms/identifier> "organization-tree".
```

Kind regards.

**Comment by josd** [comment]
Hi Sander,

Thanks for your observation and it a long standing issue which is now fixed in EYE v18.0417.2132 .
Your test now gives:
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX dcterms: <http://purl.org/dc/terms/>

<http://127.0.0.1:8080/plugins/organization-tree> rdfs:label "Organization tree"@en.
<http://127.0.0.1:8080/plugins/organization-tree> dcterms:identifier "organization-tree".

Kind regards,
Jos

**Comment by sanderzz91** [comment]
Thanks!
